controllers/istio/istio_controller.go

```diff
@@ -70,25 +73,25 @@ func NewIstioReconciler(client client.Client, scheme *runtime.Scheme, resourceDi
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.14.1/pkg/reconcile
 func (r *IstioReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	logger := log.FromContext(ctx).WithName("reconciler")
+	log := logf.FromContext(ctx)
 
 	var istio v1alpha1.Istio
 	if err := r.Client.Get(ctx, req.NamespacedName, &istio); err != nil {
 		if errors.IsNotFound(err) {
-			logger.V(2).Info("Istio not found. Skipping reconciliation")
+			log.V(2).Info("Istio not found. Skipping reconciliation")
```

Comment:
Can you add instructions somewhere (README maybe?) on how to enable verbosity on operator logs?

Reply:
Not in this PR.